apps/lib/s_cb.c

```diff
@@ -1424,8 +1424,10 @@ static int security_callback_debug(const SSL *s, const SSL_CTX *ctx,
             } else {
                 EVP_PKEY *pkey = X509_get0_pubkey(other);
                 const char *algname = "";
-                EVP_PKEY_asn1_get0_info(NULL, NULL, NULL, NULL,
-                                        &algname, EVP_PKEY_get0_asn1(pkey));
+                if (pkey != NULL) {
```

Comment:
Could you add a blank line above this.  I know it's pre-existing but worth fixing the style problem while you're here.

Reply:
Yes, I will do it soon.